### PR TITLE
Alinged the Hint in the box in fragment_send.xml

### LINKED ISSUE #1052: Redesign: Hint in TextInputEditText of fragment_send.xml is not aligned
**Summary:**

Hint in TextInputEditText of fragment_send is not aligned
**Steps to reproduce:**

Log into the app and click on payment in the bottom navigation bar

**Expected behavior:**

The Hint should be aligned in the left-center of the box

**Observed behavior:**

Capture

**Device and Android version:**

Pixel 2 API 25

### PR DESCRIPTION
**Pull request by ritish099**
## Issue Fix
Fixes #1052

## Screenshots
Screenshot_20201116-121228_Mifos Pay

## Description
Now the hints are aligned inside the box

##

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by robustTechie** [comment]
Hello @ritish099 here because of removing the text box padding the box appears to be too big can you just put the hint in the center instead of increasing box size :)

**Review by iamsh4shank — COMMENTED** [review]
Hello @ritish099 here because of removing the text box padding the box appears to be too big can you just put the hint in the center instead of increasing box size :)

**Comment by ritish099** [comment]
@robustTechie can we just make the height as "wrap content" instead of hardcoding the height. that would make it look good.
Screenshot_20201116-141113_Mifos Pay

**Comment by iamsh4shank** [comment]
Yeah you can do that, also you can try managing the padding inside the box.

**Comment by ritish099** [comment]
@robustTechie checkout

**Comment by iamsh4shank** [comment]
Looks fine to me, can you squash all the commits to one ?

**Comment by ritish099** [comment]
@robustTechie Done

**Review by iamsh4shank — CHANGES_REQUESTED** [review]
LGTM

**Review by shiv07tiwari — COMMENTED** [review]
@ritish099 Nice job. Just a couple of things:-
1. Please add the latest screenshot (after your last commit)
2. Please wait for any of the maintainer's approval on the issue before creating a Pull Request in the future.

**Comment by ritish099** [comment]
Latest Screenshot after the last commit:
error

**Review by shiv07tiwari — APPROVED** [review]

**Pull request merged**
